extensions/liquibase/runtime/src/main/java/io/quarkus/liquibase/runtime/graal/SubstituteKp.java

```diff
@@ -0,0 +1,16 @@
+package io.quarkus.liquibase.runtime.graal;
+
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+/**
+ * The liquibase license validation substitute
```

Comment:
So this comment is useless. Explain why you have a substitution instead.

Reply:
Javadoc update
The liquibase license service for the commercial liquibase license throw UnsupportedFeatureException: Non-reducible loop for the liquibase.pro.packaged.aO.parseNumberText2(boolean)